src/aiofiles/base.py

```diff
@@ -1,7 +1,6 @@
-"""Various base classes."""
+import asyncio
```

Comment:
Let's stick with directly importing (`from asyncio import X` instead of `import asyncio`). It makes the source code cleaner and it's a little faster.

Reply:
Done.